android/library/src/main/java/com/liferay/mobile/screens/util/AndroidExtensions.kt

```diff
@@ -22,7 +22,7 @@ import android.view.View
 /**
  * @author Paulo Cruz
  */
-fun TypedArray.getInt(id: Int): Int? = this.getString(id)?.toIntOrNull()
+fun TypedArray.getLong(id: Int, defValue: Long): Long = getLong(id) ?: defValue
```

Comment:
This both can be merged adding a default or null right?

Reply:
hmm, in this case, the idea was one returning optional long and the other just long. I could rename to getLongOrNull, it would be easier to understand. @victorg1991

Reply:
Yeah, you are right, I didn't notice the return time, it's fine this way :)